05/sharedtests.ml

```diff
@@ -1,5 +1,6 @@
 open Assert
 
 let shared_suite : suite =
-  [ Test ("Dbernhard tests", Dbernhard.dbernhard_tests)
+  [ Test ("Dbernhard tests", Dbernhard.dbernhard_tests);
+    Test ("thbwd tests", Thbwd.tests)
```

Comment:
I think @nicdard wants the list syntax with leading ';'

Reply:
Yes please to keep the style consistent